package.json

```diff
@@ -5,7 +5,8 @@
   "typings": "./dist/index.d.ts",
   "scripts": {
     "prepublish": "npm run build",
-    "build": "rimraf dist && tsc --pretty"
+    "build": "rimraf dist && tsc --pretty",
+    "watch": "rimraf dist && tsc --watch --pretty"
```

Comment:
Added a helper script to run tsc in watch mode for faster development.  (feel free to prune this if you don't like it)